### DISCUSSION TITLE
html_baseurl is ignored

### DISCUSSION THREAD

**Discussion by asmeurer** [pradyunsg/furo#406] [Potential Issue] [answered]
When I set the `html_baseurl` option in `conf.py`, it appears to be ignored (see https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_baseurl). I originally thought this was an issue with Sphinx, but it appears to be working correctly with the classic theme. You can test it by setting the value in conf.py and noting the configured value does not appear anywhere in the build HTML documents.

**Comment by pradyunsg** [comment]
Do you want to file a PR for this?

**Comment by asmeurer** [comment]
I'll have to see how hard it is, but I can take a look.

**Reply by pradyunsg** [reply]
Awesome, thanks! This should be a conditional tag in the base.html template, based on:

https://github.com/sphinx-doc/sphinx/blob/1e1f8e218046cbcda69be1f58234324fc48c7db4/sphinx/themes/basic/layout.html#L130
https://github.com/sphinx-doc/sphinx/blob/b12b39db05d605cfb946f84afd27e0f6cf3c9e13/sphinx/builders/html/__init__.py#L989

I missed this because it's not documented in Sphinx's templating documentation, so that's likely also something worth fixing.

**Reply by ailin-nemui** [reply]
this needs the following patch: to work correctly with dirhtml builder

https://github.com/sphinx-doc/sphinx/pull/9731

**Comment by pradyunsg** [comment, marked as answer]
This got fixed in https://github.com/pradyunsg/furo/pull/409! Thanks @asmeurer! ^>^
